crates/net/network/src/session/active.rs

```diff
@@ -860,7 +1045,7 @@ impl<N: NetworkPrimitives> OutgoingMessage<N> {
     /// Returns true if this is a response.
     const fn is_response(&self) -> bool {
         match self {
-            Self::Eth(msg) => msg.is_response(),
+            Self::Eth(msg) | Self::Snap(EthSnapMessage::Eth(msg)) => msg.is_response(),
```

Comment:
we also need `is_response` for

https://github.com/paradigmxyz/reth/blob/main/crates/net/eth-wire-types/src/snap.rs#L192-L192